### PR TITLE
adding simple fio profile for IO run and yaml file to accept input

### PR DESCRIPTION
**Pull request by Naresh-ibm**
Adding a simple fio profile which can run a quick IO without taking
much time as the default profile "fio-mixed.job" file takes.
And added a yaml file to accept the inputs like disk on which it
can run the IO and a job profile file.
altered the fiotest.py to run with yaml file.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[fio-simple-profile.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/799104/fio-simple-profile.txt)

File: perf/fiotest.py [hunk 1]
```diff
@@ -54,9 +54,10 @@ def test(self):
         """
         Execute 'fio' with appropriate parameters.
         """
+        disk = self.params.get('disk', default='')
         fio_job = self.params.get('fio_job', default='fio-mixed.job')
-        cmd = '%s/fio %s' % (self.srcdir,
-                             os.path.join(self.datadir, fio_job))
+        cmd = '%s/fio %s %s' % (self.srcdir,
```
Comment (narasimhan-v) [inline perf/fiotest.py]: @Naresh-ibm change the default profile to this one.
`fio_job = self.params.get('fio_job', default='fio-simple.job')`

File: perf/fiotest.py.data/fio-simple.job [hunk 2]
```diff
@@ -0,0 +1,73 @@
+; fio-mixed.job
```
Comment (narasimhan-v) [inline perf/fiotest.py.data/fio-simple.job]: fio-simple.job

**Comment by narasimhan-v** [comment]
Check on the PEP8 failure also.

**Comment by PraveenPenguin** [comment]
@Naresh-ibm please rebase with indent issue in here 
https://github.com/Naresh-ibm/avocado-misc-tests/blob/b45d189b7aa1b90d7af619b0585b61e7d4034049/perf/fiotest.py#L59-L60
```

        cmd = '%s/fio %s %s' % (self.srcdir,
                                                os.path.join(self.datadir, fio_job), disk)
```
like this it enable Travis Happy :)

**Comment by PraveenPenguin** [comment]
closing in  favour  of #285

**Pull request closed without merging**